clib/hMap.ml

```diff
@@ -118,6 +118,8 @@ struct
       in
       Int.Map.merge fu s1 s2
 
+  let disjoint _ _ = assert false (** Not implemented *)
```

Comment:
todo

Reply:
ok, "done" (by removing it from `USetS`).